020-quarkus-http-non-application-endpoints/src/test/java/io/quarkus/qe/non_application/endpoint/BackwardCompatibilityTest.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.qe.non_application.endpoint;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.in;
+
+
+import java.util.Collections;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.qe.non_application.endpoint.profiles.BackwardCompatibility;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+
+@QuarkusTest
+@TestProfile(BackwardCompatibility.class)
+public class BackwardCompatibilityTest extends CommonNonAppEndpoint {
+
+    @Test
+    @DisplayName("Non-application relative path")
+    public void nonAppEndpointsWithRootPathAndNonAppRootPath() {
+        for (String endpoint : nonAppEndpoints) {
```

Comment:
I would use a parametrise test here instead of this loop.

Reply:
For this scenario, I prefer it as it is.